ownCloud/Resources/de.lproj/Localizable.strings

```diff
@@ -233,6 +233,7 @@
 "Cannot connect to " = "Keine Verbindung möglich zu";
 " couldn't download this file" = "Datei konnte nicht geladen werden";
 "Actions" = "Aktionen";
+"copy" = "Kopie";
```

Comment:
@hosy Using only numbers like macOS would also remove a need to localize it

Reply:
Did you duplicate a file in Finder? For me I'm getting `Filename copy.extension` or `Filename copy 1.extension`

Reply:
@hosy hm.. yes, looks like `option + drag` and `cmd + D` do produce different results in finder